stacks-signer/src/cli.rs

```diff
@@ -152,6 +152,20 @@ pub struct GenerateFilesArgs {
     pub timeout: Option<u64>,
 }
 
+#[derive(Parser, Debug, Clone)]
+/// Arguments for the Run ommands
```

Comment:
This should be an enum.

Reply:
For what? There is a single Run command at the moment. What am I missing?